src/main/resources/page/js/home-page-loader.js

```diff
@@ -0,0 +1,35 @@
+class HomePageLoader {
```

Comment:
Could we move the home-page-related files into a new subdirectory?
js/home-page/home-page-*.js
js/home-page/ui-component-maker/ and here the header, footer and navbar files.